### PR TITLE
Fix: Fabric sort by toggles asc/desc

### LINKED ISSUE #1023: Fabric Main View and JVM/GO List of Instances View - Selecting same sort option should toggle ascending/descending order
## Expected Behavior

Selecting same sort option[Name, Status] should toggle ascending/descending sort order 

## Actual Behavior

Doesn't change ascending/descending order

## Steps to Reproduce the Problem

For Main Fabric View
  1.  On main fabric view, choose either Card or List view, and click on the same Sort option twice

For JVM/GO service list of instances view
  1.  Navigate to either JVM/GO service, and click on the same Sort option twice 

  
## Supporting Images or Videos
screen shot 2017-11-16 at 11 28 26 am

screen shot 2017-11-16 at 11 51 15 am

## Specifications

  - Version: macOS High Sierra V 10.13
  - Platform: Witnessed in Chrome 62, Safari 11.0, expecting the same behavior in all other browsers
  - Subsystem:

### PR DESCRIPTION
**Pull request by kaitmore**
Resolves #1023 

## Proposed Changes

  - Added `ascending` bool to FabricGrid state
  - Added a function to sort service statuses in the correct order: Down, Warning, Stable

## How to Test 

- **Fabric Main View => Group By: Owner/Capability/None (Card and List Views)** 
Check that Sort By Name works. Sort By Status should render in the order Down, Warning, Stable. In both case, selecting the same sort by option twice should toggle asc/desc.

### CODE REVIEW

File: src/components/Main/scenes/FabricView/components/FabricGrid/components/FabricMainView/scene/ServicesListView/ServicesListView.js [hunk 1]
```diff
@@ -64,8 +60,11 @@ function ListView({
               <ServicesList
                 items={_.orderBy(
                   dataGroupedByHeader[header],
-                  [sortByAttribute.toLowerCase(), "name"],
-                  ["asc", "asc"]
+                  item =>
+                    sortByAttribute === "Status"
+                      ? _.indexOf(microserviceStatuses, item.status)
+                      : item.name,
+                  ascending ? ["asc"] : ["desc"]
```
Comment (karinakeen) [inline src/components/Main/scenes/FabricView/components/FabricGrid/components/FabricMainView/scene/ServicesListView/ServicesListView.js]: You can add a second iteratee to sort alphabetically after you group.   ["asc", "asc"] :["desc" : "asc"]

File: src/components/Main/scenes/FabricView/components/FabricGrid/components/FabricMainView/scene/ServicesListView/ServicesListView.js [hunk 2]
```diff
@@ -82,8 +81,11 @@ function ListView({
             <ServicesList
               items={_.orderBy(
                 services,
-                [sortByAttribute.toLowerCase(), "name"],
-                ["asc", "asc"]
+                item =>
+                  sortByAttribute === "Status"
+                    ? _.indexOf(microserviceStatuses, item.status)
+                    : item.name,
+                ascending ? ["asc"] : ["desc"]
```
Comment (karinakeen) [inline src/components/Main/scenes/FabricView/components/FabricGrid/components/FabricMainView/scene/ServicesListView/ServicesListView.js]: same for others

**Comment by karinakeen** [comment]
screen shot 2017-12-12 at 9 08 09 pm

**Comment by kaitmore** [comment]
@karinakeen fixed the secondary sort so it's alphabetical. thanks!

**Pull request merged**